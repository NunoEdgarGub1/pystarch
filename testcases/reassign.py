

def f(a, b):
    c = 1
    d = 2
    c = 3

