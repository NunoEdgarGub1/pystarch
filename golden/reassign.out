testcases/reassign.py:6 reassignment "c = ..."
